pep-0454.txt

```diff
@@ -526,7 +526,7 @@ Prior Work
   <https://pypi.python.org/pypi/meliae>`_:
   Python Memory Usage Analyzer developed by John A Meinel since 2009
 * `gdb-heap <https://fedorahosted.org/gdb-heap/>`_: gdb script written in
-  Python by Dave Malcom (2010-2011) to analyze the usage of the heap memory
+  Python by Dave Malcolm (2010-2011) to analyze the usage of the heap memory
```

Comment:
Are you sure this is right? I don't know the guy and the link is dead.

Reply:
Yes, I've searched him in Google. See for example https://github.com/davidmalcolm.

Reply:
We could ping him here in GitHub, if needed.

Reply:
No, that's fine. Googling "gdb-heap dave malcom" also confirms it's the same person.